### PR TITLE
Calender assignment initial commit

### PR DESCRIPTION
**Pull request by pranav96vaidya**
@anjanaNewput please review it.

### CODE REVIEW

File: Calender/javascript/script.js [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+        },
+        {
+            name: "Satya Patidar",
+            dob: new Date("12/12/1983"),
+        }
+    ]
+}
+
+for (var i = 0; i < weeks.length; i++) {
+    var outerdiv = document.createElement('div');
```
Comment (anjanaNewput) [inline Calender/javascript/script.js]: check you naming

File: Calender/app/template/calender.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Calender</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="../../css/style.css">
+        <script src="../../javascript/script.js" async></script>
```
[the remark this answers is not in the document]
Comment (pranav96vaidya) [reply Calender/app/template/calender.html]: Solved this issue using onload function.

File: Calender/app/template/calender.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    </head>
+    <body>
+        <div class="document-container">
+            <div class="calender-container">
+                <div class="container-fluid">
+                    <div class="row">
+                        <div class="col-sm-6 head-container">
+                            <span id="leftArrowMonth" class="glyphicon glyphicon-chevron-left arrow"></span>
```
[the remark this answers is not in the document]
Comment (pranav96vaidya) [reply Calender/app/template/calender.html]: Rectified my mistakes.

File: Calender/app/template/calender.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                    <div class="row data-container">
+                        <div>
+                            <div class="calender-content" id="week-data">
+                            </div>
+                            <div id="date" class="calender-content"></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="modal fade" id="my" role="dialog">
```
[the remark this answers is not in the document]
Comment (pranav96vaidya) [reply Calender/app/template/calender.html]: used a meaningful name.

File: Calender/app/template/calender.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal">&times;</button>
+                            <h4 id="modal-head-data" class="modal-title"></h4>
+                        </div>
+                        <div class="modal-body">
+                            <img src="../../img/balloon.jpg" width="40">
+                            <p class="modal-desc"><strong>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
```
[the remark this answers is not in the document]
Comment (pranav96vaidya) [reply Calender/app/template/calender.html]: used stylings in the container.

File: Calender/javascript/script.js (hunk 1 above)
Comment (pranav96vaidya) [reply Calender/javascript/script.js]: rectified.

**Pull request merged**